article/brouillons/enzo.md

```diff
@@ -1,23 +1,149 @@
-- discours sur le formatage de l'Api qui ne permet pas de plugger directement dessus : retour critique sur la structuration technique du projet // le travail collaboratif et la réunion d'expertises diverses et externes sur le projet permet de souligner ses faiblesses, de renouer avec des choix d'implémentations fait au début de la conception technique et ainsi de davantage délimiter le projet en tant qu'objet technique = le projet 4 alimente/enrichie/actualise le projet AG / donc le hackathon n'est pas juste un moment ludique d'extension d'un projet global, c'est une occasion de dialogue critique avec un objet culturel
-  - RESTfullness ???
-  - interoperabilité de AG
-  - suitable data format
-  - travailler *en local* vs appeler une API
+  - Pendant le hackathon nous étions invités a *fetch* la data de l'API pour utiliser comme bon nous semble. La [documentation](https://github.com/EcrituresNumeriques/2022-Hackathon-Navigations/blob/main/DOC/README.md) nous invitait clairement a faire un dump local des epigrams (passages). Cela semble un peu contre intuitif sur la manière d'utiliser une API. Une contrainte qui aurait pu nous etre imposer aurait été de seulement d'accéder aux datas par requête. Mais apparement les organisateurs connaissaient les limitation de leur data et la flexibilité de leur API, nous proposant une alternative si l'on souhaitait faire de l'IA ou bien l'analytique et pas seulement de l'exposition ou bien forwarding.
+  
+### Possibilités 👀 Margot
+
+Nous avons envisagé de faire seulement un frontend pour render les different plots comme le montre ce [notebook observable HQ](https://observablehq.com/d/7de89004c95e1067?collection=@lenamk/hackathon-navigations); cependant le mixte technologique de notre groupe nous a fait partir sur une extraction et une tranformation en python. Nous condamnent a devoir faire des updates de cette sources of truth (mais qui ne devrait pas posé de problème sachant que lanthologie est un corpus fini).
```

Comment:
je propose cette version : 

Nous avions envisagé dans un premier temps procéder uniquement à un frontend pour render visibles les different plots (comme montré dans ce [notebook observable HQ](https://observablehq.com/d/7de89004c95e1067?collection=@lenamk/hackathon-navigations)); cependant la diversité technologique de notre groupe nous a réorienté vers une extraction et une tranformation en python. Obligés à faire des updates de cette sources of truth (mais qui ne devrait pas posé de problème sachant que lanthologie est un corpus fini). <!--je ne comprends pas-->
Cependant il reste possible de réaliser ce nous avons fait en full python ou javascript (dans un observable notebook with d3 ou jupyter notebook avec d3graph ou self encapsuled sans un site).
Les décisions prises par le groupe se justifient pour correspondre aux différents profils techniques du groupe, afin que chacun des membres puisse être plus à l'aise et efficace dans chacune de leurs tâches (tout en s'amusant).

Reply:
3620efe margot fixes

Reply:
3620efe margot fixes